service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -270,12 +270,11 @@ public LinkedAccountWithPassportAndVisas getRefreshedPassportsAndVisas(
   }
 
   @VisibleForTesting
-  String validatePassportWithProvider(PassportVerificationDetails passportDetails) {
-    var providerProperties =
-        externalCredsConfig.getProviders().get(passportDetails.getProviderId());
+  String validateVisaWithProvider(VisaVerificationDetails visaDetails) {
```

Comment:
I would log the response (no matter what it is) at the end of this function including the linked account id and provider id and forego the log statement on 197

Reply:
@dvoet is this change what you had in mind? https://github.com/DataBiosphere/terra-external-credentials-manager/pull/69/commits/c9f33c4ff978db47eceff91a18865f1414c13c72